src/components/graph-view-props.js

```diff
@@ -77,6 +78,7 @@ export type IGraphViewProps = {
   onSwapEdge?: (sourceNode: INode, targetNode: INode, edge: IEdge) => void,
   onUndo?: () => void,
   onUpdateNode?: (node: INode) => void,
```

Comment:
onUpdateNode should already tell you that the node moved. It passes the new X,Y coordinates within the INode type.